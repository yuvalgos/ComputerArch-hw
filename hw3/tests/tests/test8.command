./dflow_calc tests/test8.dat tests/test8.in d70 p75 d51 p102 p114 p127 p121 p198 d11 p86 d113 p82 d104 d1 p166 p20 d157 p78 d18 p129 d9 d154 d73 d156 p167 d51 d2 p59 p9 d33 p69 d146 p47 p191 d64 d141 p156 p17 d20 d145 p51 p157 p9 p149 p95 p74 p1 d196 d21 p15 p156 p158 p87 d48 d95 p199 d29 p85 d92 p122 d80 p165 p39 d190 d162 d153