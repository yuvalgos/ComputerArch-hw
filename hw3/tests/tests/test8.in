2 9 12 20
4 11 23 24
1 26 19 3
8 12 2 9
8 17 16 24
7 21 21 22
2 18 21 25
6 26 14 15
7 15 16 31
2 20 18 0
3 1 17 14
1 21 18 25
0 24 22 10
6 1 14 14
2 8 26 15
7 22 9 17
7 10 10 17
2 16 19 21
7 18 5 6
5 30 6 9
6 9 15 25
7 7 19 16
2 23 5 4
2 11 6 13
0 13 12 2
3 29 29 14
1 26 12 29
3 11 8 0
2 12 23 5
7 0 26 24
7 31 7 9
7 17 25 24
8 17 3 12
2 31 12 2
6 3 14 3
8 14 16 7
2 11 31 23
1 26 10 21
2 24 7 27
6 4 10 13
4 8 17 31
0 28 17 9
4 8 14 14
5 12 5 19
1 15 13 15
1 23 1 4
6 19 27 14
0 26 5 0
0 23 14 16
2 30 14 26
3 8 13 26
5 24 23 25
5 3 18 28
3 0 4 31
7 2 31 20
0 12 0 23
0 8 30 16
4 5 15 28
1 7 22 30
2 17 20 3
2 13 13 8
6 24 10 11
2 16 17 17
8 8 12 12
6 26 27 6
7 1 7 30
4 6 17 4
7 31 11 4
4 2 13 5
5 19 25 12
7 3 17 16
4 11 7 0
5 2 15 25
2 10 9 24
3 31 20 3
3 7 29 11
8 29 16 22
0 12 0 11
7 21 31 8
4 8 6 19
8 18 12 6
2 27 6 23
4 7 19 22
0 10 29 10
2 10 18 5
7 29 24 17
3 8 19 23
7 13 20 8
2 4 26 20
2 6 5 18
7 15 30 25
8 27 5 1
2 29 21 7
5 27 4 14
0 2 30 4
0 21 28 2
5 29 6 11
0 0 19 0
4 16 8 0
2 9 14 15
4 14 19 28
8 4 15 15
6 3 22 14
0 19 2 14
5 29 18 16
1 2 10 10
8 30 5 24
7 1 18 1
0 1 16 24
4 13 30 25
0 22 30 6
8 16 14 11
4 12 13 13
4 2 4 21
7 6 29 4
0 4 13 31
4 17 21 28
1 18 24 7
8 27 25 0
4 14 28 20
3 25 16 3
7 29 20 4
2 31 30 27
5 5 16 20
8 16 6 6
5 3 9 21
3 28 15 6
7 17 3 15
0 10 4 16
8 10 27 21
4 27 8 5
1 20 1 17
7 21 25 14
6 10 29 4
0 12 22 4
3 26 24 3
4 9 3 13
3 21 31 4
3 12 20 9
4 17 10 29
6 13 19 29
5 3 9 0
7 23 16 21
8 3 11 4
1 21 21 28
1 2 6 29
7 27 29 21
6 16 6 18
0 19 29 23
6 14 19 31
2 6 21 26
0 22 16 14
7 20 17 18
3 26 22 27
4 30 0 19
4 31 25 13
7 3 4 6
6 0 9 7
5 2 16 28
1 9 3 11
3 10 17 4
7 31 22 14
8 21 1 6
5 27 6 11
5 22 27 17
0 8 21 1
8 2 14 12
6 25 20 6
8 17 20 7
2 28 10 23
2 5 24 27
8 24 1 4
8 28 16 4
7 11 31 1
7 1 3 16
6 14 0 16
3 16 11 13
5 17 26 14
1 13 0 1
0 25 22 2
0 12 2 18
7 19 8 14
1 20 19 6
8 25 9 1
8 8 27 20
3 21 30 26
1 22 30 17
8 8 25 21
8 10 27 10
0 4 11 7
4 13 1 21
5 17 4 1
3 12 26 12
6 11 7 27
8 27 6 8
1 24 17 10
7 18 1 22
2 30 24 5
5 28 19 27
4 16 28 8
