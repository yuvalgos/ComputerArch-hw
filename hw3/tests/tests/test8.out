Reading the opcodes latency info from tests/test8.dat ... Got latency for 9 opcodes
Reading the program file tests/test8.in ... Found 200 instructions
getProgDepth()==297
getInstDeps(70)=={59,62}
getDepDepth(75)==93
getInstDeps(51)=={48,-1}
getDepDepth(102)==102
getDepDepth(114)==131
getDepDepth(127)==116
getDepDepth(121)==95
getDepDepth(198)==273
getInstDeps(11)=={6,-1}
getDepDepth(86)==91
getInstDeps(113)=={101,95}
getDepDepth(82)==91
getInstDeps(104)=={80,98}
getInstDeps(1)=={-1,-1}
getDepDepth(166)==211
getDepDepth(20)==19
getInstDeps(157)=={136,82}
getDepDepth(78)==101
getInstDeps(18)=={-1,-1}
getDepDepth(129)==116
getInstDeps(9)=={6,-1}
getInstDeps(154)=={97,148}
getInstDeps(73)=={20,61}
getInstDeps(156)=={115,150}
getDepDepth(167)==165
getInstDeps(51)=={48,-1}
getInstDeps(2)=={-1,-1}
getDepDepth(59)==51
getDepDepth(9)==24
getInstDeps(33)=={28,-1}
getDepDepth(69)==78
getInstDeps(146)=={121,144}
getDepDepth(47)==50
getDepDepth(191)==253
getInstDeps(64)=={-1,-1}
getInstDeps(141)=={136,97}
getDepDepth(156)==165
getDepDepth(17)==27
getInstDeps(20)=={8,-1}
getInstDeps(145)=={114,121}
getDepDepth(51)==76
getDepDepth(157)==127
getDepDepth(9)==24
getDepDepth(149)==194
getDepDepth(95)==81
getDepDepth(74)==80
getDepDepth(1)==0
getInstDeps(196)=={174,186}
getInstDeps(21)=={-1,17}
getDepDepth(15)==23
getDepDepth(156)==165
getDepDepth(158)==167
getDepDepth(87)==98
getInstDeps(48)=={35,17}
getInstDeps(95)=={41,94}
getDepDepth(199)==286
getInstDeps(29)=={26,12}
getDepDepth(85)==59
getInstDeps(92)=={78,82}
getDepDepth(122)==116
getInstDeps(80)=={77,66}
getDepDepth(165)==188
getDepDepth(39)==42
getInstDeps(190)=={174,185}
getInstDeps(162)=={108,150}
getInstDeps(153)=={151,146}
